src/sgame/sg_bot_util.h

```diff
@@ -34,6 +34,7 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 bool PlayersBehindBotInSpawnQueue( gentity_t *self );
 int      BotGetDefaultSkill();
 void     BotSetSkillLevel( gentity_t *self, int skill );
+std::pair<std::string, skillSet_t> BotDetermineSkills( gentity_t *bot, int skill);
```

Comment:
I don't think it makes sense to return the string and the bitset together. You can have a separate function that converts the bitset to a string (assuming the string will be used at all, given the potential issue with it being too long)

Reply:
The problem with that is that there is no straightforward skill->string mapping, and that one needs to write the code to do that. I don't think it's a big problem to keep some 50chars-long strings for each bots anyway.